### DISCUSSION TITLE
Similar app for iPad?

### DISCUSSION THREAD

**Discussion by dorrington** [MonitorControl/MonitorControl#1374] [Q&A]
Hi.

I use, and love, this on my Macs, but I'm finding myself using my iPad Pro more and more with an external monitor.  Problem is I can't find a way to have the iPad control the volume of the external monitor.  Does anyone know of something similar for iPadOS.

Cheers,
  Adrian.

**Comment by waydabber** [comment]
I don't know any I am afraid. :(
